### DISCUSSION TITLE
awattar solar feedin tarif

### DISCUSSION THREAD

**Discussion by fxstein** [mampfes/ha_epex_spot#61] [General]
Any chance to add the feed-in tarrif for solar overproduction? 

I realize its only changing monthly, but would simplify the setup of the energy dashboard and calculations if the feedin tarrif was available as part of the plugin.

Thank you!

**Comment by Vaszago** [comment]
Read this post. There is a solution here.
https://github.com/mampfes/ha_epex_spot/discussions/41

**Reply by fxstein** [reply]
Thank you. Yes saw that. Good workaround, still would be best to have this done by the HA integration.

**Reply by mampfes** [reply]
Sorry, this integration will not handle provider specific tariffs.
